Remove notify error on unsupported query for Legend Query to avoid co… (#926)
finos/legend-studio: .changeset/spicy-terms-whisper.md

---
"@finos/legend-query": patch
---

> Remove notify error on unsupported query for Legend Query to avoid confusion for execution/edition.

Reviewer: ```suggestion
Stop notifying errors for unsupported query in `Legend Query` to avoid confusion during execution/edition.
```
